.gitlab-ci.yml

```
@@ -11,15 +11,15 @@ include:
     file: '.gitlab-ci-github-status-updates.yml'
 
 trigger:mender-qa:
-  image: alpine
+  image: python:alpine
   stage: trigger
   before_script:
-    - apk add --no-cache git curl bash python3 py3-pip
+    - apk add --no-cache git curl bash
     - pip3 install pyyaml
     - git clone https://github.com/mendersoftware/integration.git integration
-    - git clone https://github.com/mendersoftware/mender-qa.git mender-qa
+    - wget -q https://raw.githubusercontent.com/mendersoftware/mender-qa/master/scripts/gitlab_trigger_client_publish && chmod +x gitlab_trigger_client_publish
```

For better readability, I suggest:
```suggestion
    - wget -q https://raw.githubusercontent.com/mendersoftware/mender-qa/master/scripts/gitlab_trigger_client_publish
    - chmod +x gitlab_trigger_client_publish
```

It is really the same, as GitLab just concatenates all the lines in one script.